pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -115,38 +115,6 @@ func getCloudCredentialSecretInfo(namespace, name string) (string, string) {
 	return namespace, name
 }
 
-func validateCreatorID(request *webhook.Request, oldCluster, cluster *v1.Cluster) *metav1.Status {
-	status := &metav1.Status{
-		Status: "Failure",
-		Reason: metav1.StatusReasonInvalid,
-		Code:   http.StatusUnprocessableEntity,
-	}
-
-	if request.Operation == admissionv1.Create {
-		// When creating the cluster the annotation must match the user creating it
-		if cluster.Annotations[auth.CreatorIDAnn] != request.UserInfo.Username {
-			status.Message = "creatorID annotation does not match user"
-			return status
-		}
-		return nil
-	}
-
-	// Check that the anno doesn't exist on the update object, the only allowed
-	// update to this field is deleting it.
-	if _, ok := cluster.Annotations[auth.CreatorIDAnn]; !ok {
-		return nil
-	}
-
-	// Compare old vs new because they need to be the same, no updates are allowed for
-	// the CreatorIDAnn
-	if oldCluster.GetAnnotations()[auth.CreatorIDAnn] != cluster.GetAnnotations()[auth.CreatorIDAnn] {
-		status.Message = "creatorID annotation cannot be changed"
-		return status
-	}
-
-	return nil
-}
-
 func (p *provisioningClusterValidator) validateClusterName(request *webhook.Request, response *webhook.Response, cluster *v1.Cluster) error {
```

Comment:
Not part of changes, but may also want to move to `validation.go`?

Reply:
That file is meant to be for common code. This code is specific to the cluster validator.